iowrappers/photos_client.go

```diff
@@ -23,6 +23,10 @@ var isHtmlAnchor = func(node *html.Node) bool {
 	return node.Type == html.ElementNode && node.Data == "a"
 }
 
+var isValidPhotoUrl = func(url string) bool {
```

Comment:
Nit: can this be `const` instead of `var`?

Reply:
I tried, but it would give a compilation error when changed to const, see leave it as "var" now. I am not fully sure how function object is treated in golang.